observability/README.md

```diff
@@ -0,0 +1,28 @@
+
+# Observability Stack
+
+This directory contains an observability implementation based on Grafana tooling
+
+## Caveats
+1) reliance on ref-implementation for SSO
+   - This is possible to work around by removing the `auth.generic_oauth` section from `prometheus.yaml` and removing the `grafana-config.yaml` and `grafana-external-secret.yaml` files
+2) using `tls_skip_verify_insecure` for oauth
+    - This is due to using the ingress certificate. Once this is addressed, we can remove this
+3) Bigger memory requirement required for kind cluster
+    - Due to using a more robust loki deployment, the memory limits have been increased. 16 GB seems to work while leaving ample room in the cluster. 
+
+## Components
+The observability stack is built upon:
+- Prometheus - metrics
+- Loki - logging
+  - Promtail - log delivery
```

Comment:
Interested in why promtail is used and something like fluentbit or opentelemetry collector isn't being used

Reply:
Promtail is really easy to use if only Loki is being used. That being said, I may look at switching to fluent bit as I've used it in another project recently. 

Is there a compelling reason to move from promtail to either? Grafana Agent could be another option as well.

Reply:
fluentbit is more popular among EKS end users also supported when using fargate

Reply:
opentelemety for logs is very new and not a lot end users have adopted

Reply:
It should be a pretty simple swap for fluebtbit. Let me take a look at.